target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -218,6 +218,9 @@ qxwlan,e750a-v4-16m)
 	ucidef_set_led_switch "lan" "LAN" "green:lan" "switch0" "0x02"
 	ucidef_set_led_netdev "wan" "WAN" "green:wan" "eth1"
 	;;
+glinet,gl-x300b)
+	ucidef_set_led_netdev "wan"   "WAN"   "green:wan" "eth1"
```

Comment:
Hmm, how could you improve this one ....?

Reply:
I thought that I had validated and cleaned up the alignment after removing the other entries, I see that I failed. I will correct this shortly.

Reply:
Well, even better would be to merge this case with another one that has the same content. You won't have to search for long ...

Reply:
Indeed, the required and corrected entry was only a line away.